dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaClusterCPUChartTask.java

```diff
@@ -61,7 +61,12 @@ protected void doRun(
             cpuPerClusterQuery,
             cluster.getName());
 
-        JsonNode chartInJson = requestTimeSeriesChart(handle, cpuPerClusterQuery);
+        JsonNode chartInJson;
+        try {
+          chartInJson = requestTimeSeriesChart(handle, cpuPerClusterQuery);
+        } catch (TimeSeriesException error) {
```

Comment:
There are `Exceptions` and `Errors` in Java. It's agreement to use `exceptions` in java code, please rename the variables `error` -> `exception`. 

additional info:
`java.lang.Error`
`java.lang.Exception`
`java.lang.RuntimeException`

Reply:
Got it, thanks!